packages/wrangler/src/api/upgrade.ts

```diff
@@ -0,0 +1,94 @@
+import { writeFileSync, existsSync } from "fs";
+import path from "path";
+import { execaSync } from "execa";
+import { fetch } from "undici";
+import { findWranglerToml } from "../config";
+import { confirm } from "../dialogs";
+import { printWranglerBanner } from "../index";
+import { logger } from "../logger";
+import { readFileSync } from "../parse";
+
+import type { PackageJSON } from "../parse";
+
+export async function upgradeWrangler({
+	yesFlag = false,
+}: {
+	yesFlag: boolean;
+}) {
+	await printWranglerBanner();
+	let packageJsonData: undefined | PackageJSON;
+
+	const pathToWorker = findWranglerToml();
+	if (pathToWorker) {
+		const newPath = pathToWorker.split(path.sep).slice(0, -1).join(path.sep);
+		packageJsonData = JSON.parse(
+			readFileSync(`${newPath}/package.json`)
+		) as PackageJSON;
+
+		if (packageJsonData.devDependencies?.wrangler) {
+			try {
+				const wranglerDistTags = (
+					(await (
+						await fetch("https://registry.npmjs.org/wrangler")
+					).json()) as {
+						"dist-tags": { latest: string };
+					}
+				)["dist-tags"];
```

Comment:
Could this be separated into multiple lines?

Reply:
Are you concerned about readability?

Reply:
Yes